### PR TITLE
Add personal userspace and maps

### PR DESCRIPTION
**Pull request by draevin**
## Description

Includes:
- Add `draevin` userspace (Hey, that's me!)
- Add keymaps for my current boards:
  - Keeb.io Sinc (80)
  - Keeb.io Quefrency (65 w/ macro)
  - DZ60

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by zvecr** [comment]
You will want to add copyright headers to all the `*.c` and `*.h` file you have added.

**Comment by draevin** [comment]
So, I was curious about that, specifically in regards to the files in the `users/draevin` directory. The [Userspace](https://beta.docs.qmk.fm/using-qmk/software-features/feature_userspace#readme-readme-md) portion of the docs specifically calls out for the license to be included in the readme but does not explicitly call for it in the other files. I did see some examples of that pattern without blocks in the associated `*.c` and `*.h` files, as well as at least one example of a separate `LICENSE` file which is a pattern I know is sometimes followed for repos. I did, however, include copyright blocks for all source files outside of `users/draevin` where they are not accompanied by the license. 

Ultimately I'm asking: is the `LICENSE` file in `users/draevin` not sufficient for the other files therein? On the other side of that, if I were to add the blocks in the associated `*.c` and `*.h` files, would that `LICENSE` or (or in line with the docs, a license in the readme) then become superfluous? I'm willing to follow whatever the established pattern is, but docs and existing examples seemed inconsistent so I was unsure. Thanks for any clarification you can provide.

File: users/draevin/rules.mk [hunk 1]
```diff
@@ -0,0 +1,12 @@
+SRC += draevin.c
+
+EXTRAFLAGS += -flto
```
Comment (fauxpark) [inline users/draevin/rules.mk:3]:
```suggestion
LTO_ENABLE = yes
```

File: users/draevin/draevin.c [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        case CM_QW: // Switch between Colemak and QWERTY
+            if (record->event.pressed) {
+                layer_invert(_CM);
+                layer_invert(_QW);
+            }
+            return false;
+        case KC_MAKE:
+            if (!record->event.pressed) {
+                SEND_STRING("make " QMK_KEYBOARD ":" QMK_KEYMAP ":flash");
+                tap_code(KC_ENT);
```
Comment (fauxpark) [inline users/draevin/draevin.c:14]:
```suggestion
                SEND_STRING("make " QMK_KEYBOARD ":" QMK_KEYMAP ":flash\n");
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]
LGTM

**Comment by noroadsleft** [comment]
> Ultimately I'm asking: is the `LICENSE` file in `users/draevin` not sufficient for the other files therein? On the other side of that, if I were to add the blocks in the associated `*.c` and `*.h` files, would that `LICENSE` or (or in line with the docs, a license in the readme) then become superfluous? I'm willing to follow whatever the established pattern is, but docs and existing examples seemed inconsistent so I was unsure. Thanks for any clarification you can provide.

I actually raised this question internally a week or two ago, and the feedback I got was that including a `LICENSE` file isn't sufficient because it's possible for the source code to become separated from the corresponding license file. Having the header in the file prevents this separation.

**Comment by draevin** [comment]
I figured that was the intent with having the header localized to the file; thanks for the clarification @noroadsleft 

I'll go ahead and add them, and assume it's fine to remove the separate `LICENSE` as it would seem redundant at that point.

**Comment by skullydazed** [comment]
For some more background GNU covers this issue in their FAQ- [Is it enough just to put a copy of the GNU GPL in my repository?](https://www.gnu.org/licenses/gpl-faq.en.html#LicenseCopyOnly) The next question is relevant as well- [Why should I put a license notice in each source file?](https://www.gnu.org/licenses/gpl-faq.en.html#NoticeInSourceFile)

File: users/draevin/rules.mk [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+AUDIO_ENABLE = no
+BACKLIGHT_ENABLE = no
+BOOTMAGIC_ENABLE = no
+CONSOLE_ENABLE = no
+SLEEP_LED_ENABLE = no
+
+MOUSEKEY_WHEEL_DELAY = 100
+MOUSEKEY_WHEEL_MAX_SPEED = 6
+MOUSEKEY_WHEEL_TIME_TO_MAX = 64
```
Comment (drashna) [inline users/draevin/rules.mk:16]: This doesn't do anything.  These need to be set in the `config.h`

```suggestion
```

File: users/draevin/draevin.h [hunk 4]
```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include QMK_KEYBOARD_H
+#include "wrappers.h"
+
+#define TAPPING_TOGGLE 2
+#define TAPPING_TERM 150
```
Comment (drashna) [inline users/draevin/draevin.h:7]: These must be in the `config.h`
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**